docs/manifest.json

```diff
@@ -134,6 +134,18 @@
       "icon_path": "./images/icons/users.svg",
       "path": "./users.md"
     },
+    {
+      "title": "Admin",
+      "description": "How to install and deploy Coder",
+      "icon_path": "./images/icons/cog.svg",
```

Comment:
🚧 `"icon_path": "./images/icons/cog.svg",` 

I'm not sure (yet) where `/images/icons` is being imported from and picked `cog.svg` to match the pattern from Coder v1 documentation. Will need changing as not sure if `cog.svg` is even a thing...

<img width="145" alt="CleanShot 2022-09-07 at 12 33 55@2x" src="https://user-images.githubusercontent.com/127353/188775736-ce677115-f61e-42ff-aa8e-c5747c62c25e.png">